build/source/dshare/data_types.f90

```diff
@@ -0,0 +1,140 @@
+! SUMMA - Structure for Unifying Multiple Modeling Alternatives
+! Copyright (C) 2014-2015 NCAR/RAL
+!
+! This file is part of SUMMA
+!
+! For more information see: http://www.ral.ucar.edu/projects/summa
+!
+! This program is free software: you can redistribute it and/or modify
+! it under the terms of the GNU General Public License as published by
+! the Free Software Foundation, either version 3 of the License, or
+! (at your option) any later version.
+!
+! This program is distributed in the hope that it will be useful,
+! but WITHOUT ANY WARRANTY; without even the implied warranty of
+! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+! GNU General Public License for more details.
+!
+! You should have received a copy of the GNU General Public License
+! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+MODULE data_types
+ ! used to define model data structures
+ USE nrtype
+ USE multiconst,only:integerMissing
+ implicit none
+ private
+
+ ! ***********************************************************************************************************
+ ! Define the model decisions
+ ! ***********************************************************************************************************
+ ! the model decision structure
+ type,public  :: model_options
+  character(len=64)                      :: cOption='notPopulatedYet'
```

Comment:
No - I simply meant to set `64` to some constant. Also to make sure that we are consistent in the use of missing values in various parts of the code, but it looked fine to me here.